### PR TITLE
Add Remote Book dialog smoke test

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Adds simple smoke test for Add Remote Book dialog

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "azuredatastudio",
   "version": "1.31.0",
-  "distro": "462bd5082b18c2ed5df4ba615d3dda9a2860a394",
+  "distro": "d69a2de3200adff1ccbccd53a6dae7074f883bb5",
```
Comment (kburtram) [inline package.json:4]: I've added some changes to the release branch for web build that you'll likely need to pickup to get a clean build.  I think the commit hash `9eab1e6d6180c1a560a4a50eb9e6e99641549329` should be fine.

Comment (Charles-Gagnon) [reply package.json:4]: Thanks, sorry about causing you problems with the web mode stuff - I want to follow up and figure out if we can make it so we don't have to update the distro when adding new smoke tests if I can to hopefully avoid this kind of thing in the future!

File: test/smoke/src/sql/areas/notebook/createBook.test.ts [hunk 2]
```diff
@@ -61,6 +61,9 @@ export function setup() {
 					// Try our best to clean up but don't fail the test if we can't
 				}
 			}
+			const app = this.app as Application;
+			// Workaround for error notification trying to read deleted books
```
Comment (Charles-Gagnon) [inline test/smoke/src/sql/areas/notebook/createBook.test.ts:65]: I was hitting issues with the toasts from these tests blocking the dialog. It's something I plan on coming back to fix in a better way but hopefully this should at least prevent issues in the near term.

**Comment by Charles-Gagnon** [comment]
I'm going to merge this in to make sure the distro bump doesn't cause any more problems, but it seems like I might need to follow up with some more work on stuff blocking UI elements for other tests.

**Pull request merged**